Rakefile

```diff
@@ -62,21 +62,16 @@ namespace :pypi do
 
   task :build => :clean do
     puts "building release in #{RELEASE_DIR}"
-    # TODO: Use `scripts/build-dist` instead to build sdist and wheels
-    sh "python setup.py -q sdist -d #{RELEASE_DIR}"
+    sh "scripts/build-dist"
   end
 
   task :release => [:confirm, :install, :build] do
     builds = Dir.entries(RELEASE_DIR).reject {|f| f == '.' || f == '..'}
     if builds.length == 0
         fail "no build found in #{RELEASE_DIR}"
-    elsif builds.length > 1
-        fail "multiple builds found in #{RELEASE_DIR}"
     end
 
-    build = "#{RELEASE_DIR}/#{builds[0]}"
-
-    puts "uploading #{build}"
-    sh "twine upload #{build}"
+    puts "uploading #{RELEASE_DIR}/*"
+    sh "twine upload #{RELEASE_DIR}/*"
```

Comment:
Do we need to make sure that the RELEASE_DIR was empty before building and uploading?
I'm afraid we'd upload old/stale build.